designer/client/component-create.js

```diff
@@ -1,37 +1,31 @@
 import React from 'react'
-import { getFormData } from './helpers'
 import ComponentTypeEdit from './component-type-edit'
 import ComponentTypes from '@xgovformbuilder/model/lib/component-types'
 import { clone } from '@xgovformbuilder/model/lib/helpers'
 
 class ComponentCreate extends React.Component {
-  state = {}
+  constructor (props) {
+    super(props)
+    this.state = {}
+    this.storeComponent = this.storeComponent.bind(this)
```

Comment:
I figured out what the problem was with this while reading up on class members. I don't think you need to bind if it's an arrow function. (Arrow functions are bound in the constructor) so `onSubmit = (e) => { }` should work 🤞

Reply:
nice! Thanks!